lib/ansible/modules/packaging/os/yum.py

```diff
@@ -114,6 +114,13 @@
     type: bool
     default: "no"
     version_added: "2.4"
+  bugfix:
+    description:
+      - If set to C(yes), and C(state=latest) then only installs updates that have been marked bugfix related.
+    required: false
+    default: "no"
+    choices: ["yes", "no"]
```

Comment:
Listing `choices` for a boolean type is no longer necessary https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_checklist.html